sdk/storage/storage-blob/src/generatedModels.ts

```diff
@@ -153,3 +149,18 @@ export {
   ContainerRenameResponse,
   ContainerRenameHeaders
 } from "./generated/src/models";
+
+import {
+  BlobHttpHeaders,
+  BlobSetHttpHeadersResponse,
+  BlobSetHttpHeadersHeaders
+} from "./generated/src/models";
+
+/** Contains response data for the setHttpHeaders operation. */
```

Comment:
should we comment these as legacy compat names?

Reply:
Updated this to be code like:
```
export {
  BlobSetHttpHeadersResponse as BlobSetHTTPHeadersResponse 
} ...
```